modules/mbedtls/Kconfig.mbedtls

```diff
@@ -257,15 +262,13 @@ config MBEDTLS_CIPHER_ALL_ENABLED
 	select MBEDTLS_CHACHAPOLY_AEAD_ENABLED
 
 config MBEDTLS_SOME_AEAD_CIPHER_ENABLED
-	bool
-	default y
+	def_bool y
```

Comment:
Minor comment.
While the change is technically OK, the scope of the commit is different so it would have been better to place it in a separate commits IMO. The same holds also for `MBEDTLS_SOME_CIPHER_ENABLED` and `MBEDTLS_PSA_CRYPTO_CLIENT`.